README.md

```diff
@@ -0,0 +1,48 @@
+# AdditionalFruits
+A set of fruits that grow using the SimpleFarming module. For more information, check out the [SimpleFarming Module](https://github.com/Terasology/SimpleFarming).
+
+## Contains:
+ * Blueberry
+ * Cranberry
+ * Gojiberry
+ * Gooseberry
+ * Guava
+ * Melon
+ * Passionfruit
+ * Peach
+ * Pineapple
+ * Raspberry
+ * Strawberry
+ * Tomato
+
```

Comment:
A contributing section would be nice as well. Doesn't need to be long, just mentioning that this module is for fruits, a link to SimpleFarming readme on how to add a plant and telling the reader to open a pull request with the new content.